TESTS/mbed_functional/callback/main.cpp

```diff
@@ -695,12 +695,28 @@ utest::v1::status_t test_setup(const size_t number_of_cases)
 }
 
 Case cases[] = {
+#ifdef DO_BIG_TEST
```

Comment:
How is that decided?

Reply:
That's set when `callback/main.cpp` is included from `callback_big/main.cpp`.

It's just a break-up of the test set into 3 images (callback/callback_big/callback_small). Breakup was done because so many template instantiations didn't fit into a tiny target.

Original split was done by copy-paste, but the 3 tests had started to diverge.

See commit https://github.com/ARMmbed/mbed-os/pull/12035/commits/8b9f33a0fe3a9b47d8497d815828a3a920733412